### ISSUE TITLE
Spec build fails with "cannot load such file -- rexml/parsers/baseparser"

### ISSUE THREAD

**Issue by SethTisue** [scala/scala-dev#832]
seen at e.g. https://app.travis-ci.com/github/scala/scala/jobs/595963138

```
$ bundle exec jekyll build -s spec/ -d build/spec
/home/travis/.rvm/gems/ruby-3.1.2/gems/kramdown-1.17.0/lib/kramdown/parser/html.rb:10:in `require': cannot load such file -- rexml/parsers/baseparser (LoadError)
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/kramdown-1.17.0/lib/kramdown/parser/html.rb:10:in `<top (required)>'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/kramdown-1.17.0/lib/kramdown/parser/kramdown/html.rb:10:in `require'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/kramdown-1.17.0/lib/kramdown/parser/kramdown/html.rb:10:in `<top (required)>'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/kramdown-1.17.0/lib/kramdown/parser/kramdown/paragraph.rb:14:in `require'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/kramdown-1.17.0/lib/kramdown/parser/kramdown/paragraph.rb:14:in `<top (required)>'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/kramdown-1.17.0/lib/kramdown/parser/kramdown.rb:345:in `require'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/kramdown-1.17.0/lib/kramdown/parser/kramdown.rb:345:in `<class:Kramdown>'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/kramdown-1.17.0/lib/kramdown/parser/kramdown.rb:60:in `<module:Parser>'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/kramdown-1.17.0/lib/kramdown/parser/kramdown.rb:19:in `<module:Kramdown>'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/kramdown-1.17.0/lib/kramdown/parser/kramdown.rb:17:in `<top (required)>'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/jekyll-3.6.3/lib/jekyll/converters/smartypants.rb:3:in `require'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/jekyll-3.6.3/lib/jekyll/converters/smartypants.rb:3:in `<top (required)>'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/jekyll-3.6.3/lib/jekyll.rb:13:in `require'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/jekyll-3.6.3/lib/jekyll.rb:13:in `block in require_all'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/jekyll-3.6.3/lib/jekyll.rb:12:in `each'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/jekyll-3.6.3/lib/jekyll.rb:12:in `require_all'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/jekyll-3.6.3/lib/jekyll.rb:189:in `<top (required)>'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/jekyll-3.6.3/exe/jekyll:8:in `require'
	from /home/travis/.rvm/gems/ruby-3.1.2/gems/jekyll-3.6.3/exe/jekyll:8:in `<top (required)>'
	from /home/travis/.rvm/gems/ruby-3.1.2/bin/jekyll:25:in `load'
	from /home/travis/.rvm/gems/ruby-3.1.2/bin/jekyll:25:in `<main>'
	from /home/travis/.rvm/gems/ruby-3.1.2/bin/ruby_executable_hooks:22:in `eval'
	from /home/travis/.rvm/gems/ruby-3.1.2/bin/ruby_executable_hooks:22:in `<main>'
```

**Comment by SethTisue** [comment]
so far I've only seen this on the one PR — not sure what to make of that

**Comment by SethTisue** [comment]
Optimistically closing on the hopes that https://github.com/scala/scala/pull/10312 will fix it on 2.12.x and https://github.com/scala/scala/pull/10310 will fix it on 2.13.x.

We can decide later whether we want to bring the branches back in sync. On 2.12.x we pinned an older Ruby version ; on 2.13.x we moved to newer jekyll and redcarpet versions; may the best branch win!
